src/main/resources/application.yml

```
@@ -0,0 +1,24 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/deviceauthorizer
+    username: radarcns
+    password: radarcns
+    driver-class-name: org.postgresql.Driver
+
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQL94Dialect
+    properties.hibernate.temp.use_jdbc_metadata_defaults: false
+
+  jackson:
+    serialization:
+      write_dates_as_timestamps: false
+
+device-authorizer:
```

```suggestion
radar-rest-source-authorizer:
```
Is this cors config still used?